dtfs-central-api/api-tests/v1/tfm/deal-cancellation/tfm-submit-deal-cancellation-post.api-test.ts

```diff
@@ -126,9 +138,9 @@ describe('/v1/tfm/deals/:dealId/cancellation/submit', () => {
         const submitCancellationResponse = await testApi.post({ cancellation, auditDetails }).to(submitDealCancellationUrl);
 
         expect(submitCancellationResponse.body).toEqual({
-          cancelledDeal: expect.objectContaining({ _id: dealId }) as TfmDeal,
+          cancelledDealUkefId: ukefDealId,
```

Comment:
Can we add in the PR description why it's changed to return only an ID instead of the whole cancelled deal please?

Reply:
I added
> Modify `TfmDealCancellationResponse` type back to a previous version & update controllers/tests to handle this
This was done as I realised the previous changes Id made werent entirely necessary & it made more sense to decouple the Repository response type from the DTO sent to tfm-api from the controller.